app/javascript/components/notification-message.jsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { NotificationTypes } from '../helpers/notification-types';
 
-export const NotificationTypes = {
-  info: 'info',
-  danger: 'danger',
-  success: 'success',
-  warning: 'warning',
-};
-
-const NotificationMessage = ({
-  type, message,
-}) => {
-  const valid = Object.values(NotificationTypes).includes(type);
-
+/** Component to render a notification message. */
+const NotificationMessage = ({ type, message }) => {
+  const valid = Object.keys(NotificationTypes).includes(type);
   if (valid && message) {
+    const notification = NotificationTypes[type];
```

Comment:
I'm not sure we should just silently ignore messages with invalid types.  Probably better to have a type called "unknown" (or something to that effect), and then check for the given type and fall back to the unknown type.

Reply:
ok. will render the `unknown` type in the `info` (gray) notifications.

Reply:
What about gray color, but a ? icon (I think patternfly has a question mark in a circle - possibly `fa-regular fa-circle-question`

Reply:
Added the `fa-regular fa-question-circle` for unknown notification type

![](https://user-images.githubusercontent.com/87487049/260366796-68475cf9-d5d3-47f4-916e-cb9a0a3e0df1.png)